playbooks/example-use/group_vars/all

```diff
@@ -12,8 +12,7 @@ pulp_install_plugins:
   # pulp-maven: {}
   # pulp-npm: {}
   # pulp-python: {}
-  # pulp-rpm:
-  #   prereq_role: "pulp.pulp_rpm_prerequisites" # RPM needs a prereq_role: https://galaxy.ansible.com/pulp/pulp_rpm_prerequisites
+  pulp-rpm: {}
```

Comment:
So this playbook is actually meant for users to run. It is also used in molecule CI. If we want molecule CI to use it, it should be copied to molecule/release-static/group_vars/all (and symlinked from other molecule tests) rather than symlinked into molecule folders.

Reply:
I see you created the new copies of the vars files under molecule/, but now you need to restore these example-use/ vars files to their original values (except for removing `prereq_role`).